README.md

```diff
@@ -89,6 +90,27 @@ Pluggable features are defined in configuration and are disabled by default. Lis
  - `balance_data_size` - Swap partitions one by one by one if imbalance in size on brokers is bigger than 
  `FREE_SPACE_DIFF_THRESHOLD_MB` megabytes.
  
+## <a name="startup_timeout"></a> Timeouts for startup
+ Each time when bubuku tries to start kafka, it uses special startup timeout. This means, that if kafka broker id 
+ is not found within this timeout in zookeeper node `/broker/ids/{id}`, kafka process will be forcibly killed, timeout 
+ for start updated, and startup will be retried. 
+  
+  There are two ways to increase timeout - linear and progressive. Linear adds the same amount of time after each 
+  failed start. Progressive adds time, that is relative to current timeout. Configuration for that is provided by 
+  `STARTUP_TIMEOUT` parameter.
+  ```
+  # Linear timeout configuration 
+  # initial timeout=300 seconds, after each failed start increase by 60 seconds (360, 420 and so on)
+  export STARTUP_TIMEOUT="type=linear:initial=300:step=60"
```

Comment:
Nut you can just keep the name STARTUP_TIMEOUT_STEP for both modes. 

I think having some custom complex structure for parameter value is very bad. I think the only thing that is acceptable is a list of items separate by ",".
And having parameters that depend on each other is fine for me, we can't really avoid that. At least they have the same prefix "STARTUP_TIMEOUT_"